src/docs/design-toolkit/design-files/index.md

```diff
@@ -9,18 +9,27 @@ published: true
 
 ![FTW in Sketch](./FTW-in-Sketch.png)
 
+
 Saunatime Flex Template for Web (FTW) designs are available as Sketch
 files. The design files include complete designs for FTW desktop and
 mobile, as well as wireframes for FTW desktop. You can use these files
 as the basis for your own designs. This approach can dramatically speed
 up the design process, and it offers an easy starting point for the
 developers.
 
-Sharetribe offers two templates: one for daily bookings (e.g. booking
-saunas or hiking tours) and one for hourly bookings (e.g. booking yoga
-lessons or renting kayaks). The main difference between these templates
-is in the booking calendar, but also the outlooks differ to show some of
-the flexibility that Flex offers.
+## Wireframes
+
+Wireframes are an excellent tool to design your marketplace at the structural level. They are most often used to arrange content and functionality and establish the basic structure of a page before moving on to designing visual elements. Wireframes for our desktop layout are available as Sketch files below.
+
+## Design files
+
+Sharetribe offers two templates: one for daily bookings (e.g. booking saunas or hiking tours) and one for hourly bookings (e.g. booking yoga lessons or renting kayaks). The main difference between these templates is in the booking calendar, but also the outlooks differ to show some of the flexibility that Flex offers. Both templates are provided in desktop and mobile layouts. The designs are provided as .sketch files, which can be opened using the [Sketch vector graphics editor](https://www.sketch.com/). Some other design tools such as Adobe XD and Photoshop also offer support for designs in the .sketch file format.
+
+Once you have the design files installed, you can start working on customising the design to match the visual identity of your marketplace. Not only should you concentrate on how your marketplace should look like, but also on what kind of user interfaces are required for your users to interact with each other. 
```

Comment:
A minor style comment: IIRC, we've mostly used American spelling in Flex content so here _customising_ could be replaced with _customizing_.